pkgs/checks/lib/src/extensions/math.dart

```diff
@@ -86,3 +86,47 @@ extension NumChecks on Subject<num> {
     });
   }
 }
+
+extension ComparableChecks<T> on Subject<Comparable<T>> {
```

Comment:
Meh...it's almost worth it's only library. It's more than math. But it's not a biggy. 🤷

Reply:
Core actually feels morea appropriate

Reply:
Sure, I'll leave it back in `core.dart` for now. It doesn't make a user facing difference, so we can shuffle things later if we want.